src/main/ts/keepachangelog/Changelog.ts

```diff
@@ -0,0 +1,87 @@
+import { pipe } from 'fp-ts/function';
+import * as O from 'fp-ts/Option';
+import * as KeepAChangelog from 'keep-a-changelog';
+import { Semver } from 'keep-a-changelog/types/src/deps';
+import { DateTime } from 'luxon';
+import * as Version from '../core/Version';
+import * as Files from '../utils/Files';
+
+export interface Changelog extends KeepAChangelog.Changelog {
+  releases: CustomRelease[];
```

Comment:
I should add I specifically didn't make them readonly since the original from `keep-a-changelog` was not readonly either.